src/renderer/account/components/TransactionsPanel/Send/Send.js

```diff
@@ -45,49 +44,46 @@ export default class SendPanel extends React.PureComponent {
     const symbol = this.getSymbol();
 
     return (
-      <Panel className={classNames(styles.sendPanel, className)}>
-        <form className={styles.content}>
-          <h2>Send</h2>
-          <Select
-            className={styles.asset}
-            id="asset"
-            label="Token to send"
-            value={asset}
-            onChange={this.handleChangeAsset}
-          >
-            {this.renderAssets()}
-          </Select>
-          <Input
-            className={styles.amount}
-            id="amount"
-            type="number"
-            label="Sending amount"
-            placeholder={`Enter ${symbol} amount`}
-            min="0"
-            step={step}
-            value={amount}
-            onChange={this.handleChangeAmount}
-          />
-          <Input
-            className={styles.recipient}
-            id="recipient"
-            type="text"
-            label="Recipient"
-            placeholder="Wallet address"
-            value={receiver}
-            disabled={false}
-            onChange={this.handleChangeRecipient}
-          />
-          <Button
-            className={styles.next}
-            type="submit"
-            disabled={loading || !this.isValid()}
-            onClick={this.handleTransfer}
-          >
-            Next
-          </Button>
-        </form>
-      </Panel>
+      <form className={classNames(styles.send, className)}>
+        <Select
+          className={styles.asset}
+          id="asset"
+          label="Token to send"
+          value={asset}
+          onChange={this.handleChangeAsset}
+        >
+          {this.renderAssets()}
```

Comment:
Following the design, the default value - if no tokens are "found" should be `No tokens available`

Reply:
Yeah, I was planning to tackle that in another PR.  This didn't feel like the time to do it since I was trying to work on the receive tab for this PR.